source/manifest.json

```diff
@@ -15,7 +15,15 @@
 		"128": "icon.png"
 	},
 	"permissions": [
-		"storage"
+		"storage",
+		"<all_urls>"
```

Comment:
Chrome Web Store (where this extension is automatically published to) would give us a hard time if we do this, requiring justification for every permission we add.

Maybe ask for access to just this repo URL or something along those lines.